webapp/chart.js

```diff
@@ -799,7 +800,23 @@ function chart_init(graph, wnd, affichwnd, with_interval, with_interval2, show_i
                 points_list[index].enqueue([line_id, x, y]);
             }else{
                 var values = get_points_for_data(data, data[8], data[10]);
-                points_list[index].enqueue([line_id, values]);
+                cmatview_counter++; // to count lines from log
+                if(cmatview_counter == 1){
+                     //Only add points of line 1, so that no delay in chart appearance)
+                     points_list[index].enqueue([line_id, values]);
+                }else if(cmatview_counter < 16){
```

Comment:
```suggestion
                } else if (cmatview_counter[index] <= 125) {
```
Try increasing to 125. If slow. make it around 50. Also, add index as mentioned before.